### DISCUSSION TITLE
How can I open swagger document page? （我如何才能打开这个项目的swagger文档）

### DISCUSSION THREAD

**Discussion by Jessica19950202** [kubeedge/kubeedge#5912] [Q&A]
Could you please let me know if this project provides a Swagger document to view the API parameters for easier debugging and better understanding of the functionalities?

请问一下，我想要打开一个swagger文档查看API的出入参数，便于调试接口和理解功能，这个项目有提供这样的swagger文档吗？

**Comment by Shelley-BaoYue** [comment]
https://github.com/kubeedge/kubeedge/tree/master/staging/src/github.com/kubeedge/api/apidoc/generated/swagger
